R/data.R

```diff
@@ -99,3 +99,20 @@ NULL
 #' levels(hpc_cv$obs)
 NULL
 
+#' Survival Analysis Results
+#'
+#' @details This data contain plausible results, when applying predictive
+#' survival models to the [lung] data set using the censored package.
+#'
```

Comment:
Would the `prob_censored` column in this dataset be unusual to find in output from these models? It seems that took some serious elbow grease to calculate, but perhaps it will be easier with the upcoming tidymodels machinery. If it is unusual, should we document what that is + how it's calculated?

Reply:
That also means that I'll have to settle on a naming convention for the weighting column.

Reply:
> We should convert that column to the inverse probability weight 

It was the inverse probability weight, i have changed the name to better reflect it